shopinvader_api_cart/routers/cart.py

```diff
@@ -52,6 +52,21 @@ def sync(
     return Sale.from_sale_order(cart) if cart else None
 
 
+@cart_router.post("/update")
+@cart_router.post("/update/{uuid}")
+def update(
+    data: CartUpdateInput,
+    env: Annotated[api.Environment, Depends(authenticated_partner_env)],
+    partner: Annotated["ResPartner", Depends(authenticated_partner)],
+    uuid: str | None = None,
+) -> Sale:
+    cart = env["sale.order"]._find_open_cart(partner.id, uuid)
+    if not cart:
+        cart = env["sale.order"]._create_empty_cart()
+    cart.write(data.to_vals())
+    return Sale.from_sale_order(cart)
```

Comment:
@sebastienbeau This part of the code should got into an helper model as it's done for the sync route.

Reply:
ok